model/ticket.php

```diff
@@ -16,6 +17,7 @@ class Ticket implements Subject
     public $service;
     public $author;
     public $comments;
+    public $attachment;
```

Comment:
A ticket may have multiple attachments, right? so perhaps a plural name would be better (`public $attachments`).